### PR TITLE
Documentation/admin/ldap-user-management.md: give procedure flow to the doc

### PR DESCRIPTION
**Pull request by radhikapc**
- performed minor edits for clarity
- added procedural flow

### CODE REVIEW

File: Documentation/admin/ldap-user-management.md [hunk 1]
```diff
@@ -6,141 +6,157 @@ Tectonic Identity is an authentication service for both Tectonic Console and `ku
...
 * (Optional) The root CA if certificate CA verification is desired
 
-Follow these steps in the Tectonic UI to enable LDAP in your cluster:
-
-First navigate to *Cluster Settings* under *Administration*.
-
-Next click the *LDAP* link and add your LDAP host. Specify whether your LDAP host uses SSL; if so, specify whether the certificate should be verified against a trusted root CA and click *Continue*. Add an LDAP service account capable of querying for groups and users and click *Continue*.
+Follow these steps in the Tectonic UI to enable LDAP authentication in your cluster:
+
+1. On the Tectonic UI, navigate to *Cluster Settings* under *Administration*.
```
Comment (joshix) [inline Documentation/admin/ldap-user-management.md]: s/On the Tectonic UI/In Tectonic Console/

File: Documentation/admin/ldap-user-management.md [hunk 2]
```diff
@@ -6,141 +6,157 @@ Tectonic Identity is an authentication service for both Tectonic Console and `ku
...
-  <div class="col-lg-10 col-lg-offset-1 col-md-10 col-md-offset-1 col-sm-10 col-sm-offset-1 col-xs-12 col-xs-offset-1">
-    <a href="../img/ldap-server-group-search-diagram.png" class="co-m-screenshot">
-      <img src="../img/ldap-server-group-search-diagram.png">
-    </a>
-  </div>
-</div>
+  In this example the user `john.doe`'s `dn` is in `tstgrp`, but `jane.doe` is in no groups. You can query for these users in the *Test Configuration* page to verify this.
 
-Input a valid user and click *Test Configuration* to verify that users and groups are correctly configured. *If the query does not return expected results, see the [Troubleshooting](#troubleshooting) section, below.*
+9. From the *Test Configuration* page, click *Continue* to see    instructions on updating the given Tectonic Identity.
```
Comment (joshix) [inline Documentation/admin/ldap-user-management.md]: delete extra tab or spaces after "see"

File: Documentation/admin/ldap-user-management.md [hunk 3]
```diff
@@ -6,141 +6,157 @@ Tectonic Identity is an authentication service for both Tectonic Console and `ku
...
+  configmap "tectonic-identity" configured
+  ```
 
-```bash
-$ kubectl apply -f ~/Downloads/new-tectonic-config.yaml
+2. Restart the Tectonic Identity pods for the changes to take effect.
 
-configmap "tectonic-identity" configured
-```
+3. Run the following command to trigger a rolling update and attache the current date as an annotation. Verify the pods are patched.
```
Comment (joshix) [inline Documentation/admin/ldap-user-management.md]: s/attache/attach/

Comment (joshix) [inline Documentation/admin/ldap-user-management.md]: Aren't steps 2 and 3 saying the same thing? Should they be combined?

Also, "Verify the pods are patched." seems to belong _after_ the `kubectl patch` command below. And once moved there, it just repeats what the reader is told. So I think  "Verify the pods are patched" could be deleted.

File: Documentation/admin/ldap-user-management.md [hunk 4]
```diff
@@ -6,141 +6,157 @@ Tectonic Identity is an authentication service for both Tectonic Console and `ku
...
+  If successful the following message is displayed:
 
-"tectonic-identity" patched
-```
+  ```
+  "tectonic-identity" patched
+  ```
 
-You can now log out of Tectonic Console and log back in using LDAP.
+4. Log out of Tectonic Console and log in back by using the LDAP authentication credentials.
```
Comment (joshix) [inline Documentation/admin/ldap-user-management.md]: s/log in back/log in again/

File: Documentation/admin/ldap-user-management.md [hunk 5]
```diff
@@ -159,72 +175,76 @@ To use `kubectl` as an LDAP user:
 
 Until otherwise modified, you can still use your static account for further administrative setup.
 
-## Configuring access
+## Configuring Role Based Access
```
Comment (joshix) [inline Documentation/admin/ldap-user-management.md]: replace with: "Configuring Role Based Access Control (RBAC)"

File: Documentation/admin/ldap-user-management.md [hunk 6]
```diff
@@ -6,141 +6,154 @@ Tectonic Identity is an authentication service for both Tectonic Console and `ku
...
-First navigate to *Cluster Settings* under *Administration*.
-
-Next click the *LDAP* link and add your LDAP host. Specify whether your LDAP host uses SSL; if so, specify whether the certificate should be verified against a trusted root CA and click *Continue*. Add an LDAP service account capable of querying for groups and users and click *Continue*.
-
-<div class="row">
-  <div class="col-lg-10 col-lg-offset-1 col-md-10 col-md-offset-1 col-sm-10 col-sm-offset-1 col-xs-10 col-xs-offset-1">
-    <a href="../img/ldap-query-user.png" class="co-m-screenshot">
-      <img src="../img/ldap-query-user.png" class="img-responsive">
-    </a>
+Follow these steps in the Tectonic UI to enable LDAP authentication in your cluster:
```
Comment (joshix) [inline Documentation/admin/ldap-user-management.md]: s/in the Tectonic UI/in Tectonic Console/

File: Documentation/admin/ldap-user-management.md [hunk 7]
```diff
@@ -6,141 +6,154 @@ Tectonic Identity is an authentication service for both Tectonic Console and `ku
...
+
+1. In the Tectonic UI, navigate to *Cluster Settings* under *Administration*.
```
Comment (joshix) [inline Documentation/admin/ldap-user-management.md]: s/the Tectonic UI/Tectonic Console/

File: Documentation/admin/ldap-user-management.md [hunk 8]
```diff
@@ -6,141 +6,154 @@ Tectonic Identity is an authentication service for both Tectonic Console and `ku
...
+      --namespace tectonic-system
+  ```
+If successful the following message is displayed:
 
-You can now log out of Tectonic Console and log back in using LDAP.
+  ```
+  "tectonic-identity" patched
+  ```
+
+3. Log out of Tectonic Console and log in back by using the LDAP authentication credentials.
```
Comment (joshix) [inline Documentation/admin/ldap-user-management.md]: s/log in back/log in again/

File: Documentation/admin/ldap-user-management.md [hunk 9]
```diff
@@ -6,141 +6,154 @@ Tectonic Identity is an authentication service for both Tectonic Console and `ku
 
 This document describes managing users and access control in Tectonic and Kubernetes using LDAP.
 
-## Identity LDAP configuration
+## Configuring Tectonic Identity for LDAP authentication
 
-Integrating Tectonic Identity with LDAP is configured through the Tectonic console. In order to integrate with LDAP, you'll need the following:
+Tectonic Identity is configured through the Tectonic console to allow for LDAP user authentication. In order to integrate with an LDAP server, you'll need the following:
```
Comment (joshix) [inline Documentation/admin/ldap-user-management.md]: s/the Tectonic console/Tectonic Console/ (sorry for missing this)

**Comment by radhikapc** [comment]
can we merge this ? have fixed all the comments.

**Pull request merged**